applications/nrf_desktop/doc/usb_state.rst

```diff
@@ -6,3 +6,66 @@ USB state module
 The ``usb_state`` module is responsible for tracking the USB connection.
 It is also responsible for the data transmission that application does on this
```

Comment:
`for the data transmission over the USB` ?

Reply:
done